lxd/container_lxc.go

```
@@ -1852,10 +1836,7 @@ func (c *containerLXC) OnStop(target string) error {
 		deviceTaskSchedulerTrigger("container", c.name, "stopped")
 
 		// Record current state
-		err = dbContainerSetState(c.daemon.db, c.id, "STOPPED")
-		if err != nil {
-			return
-		}
+		dbContainerSetState(c.daemon.db, c.id, "STOPPED")
```

Are we sure we want to ignore the error here? Should we at least log it?